lib/skate/telemetry.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Skate.Telemetry do
+  @moduledoc """
+  Telemetry listeners for Skate business logic.
+  """
+  require Logger
+
+  def setup_telemetry do
+    _ =
+      :telemetry.attach(
+        "skate",
+        [:skate, :repo, :query],
+        &Skate.Telemetry.handle_event/4,
+        %{}
+      )
```

Comment:
nit(non-blocking): not doing anything with this assignment/match
```suggestion
    :telemetry.attach(
      "skate",
      [:skate, :repo, :query],
      &Skate.Telemetry.handle_event/4,
      %{}
    )
```